### ISSUE TITLE
Is this project still in development?

### ISSUE THREAD

**Issue by RocksteadyTC** [turtlecoin/krang#1]
Curious

**Comment by SoreGums** [comment]
it is. i keep rearranging what I'm doing so the initial commit is lagging.

too many ideas, little focus.
so spinning wheels right now, however this will materialise this week.

- a map/overview 
- docker image builder for turtlecoind
  - this is the beginning step, once can spin up random compilations or tweaked settings then can start honing in on the point of this project.

**Comment by 4Reallive** [comment]
@RocksteadyTC Yeah it still is. i started a new job. Taking some time to do things right, will make a commit in the next couple of weeks! Also spoke with funky penguin, will integrate with his docker swarm pool once this starts moving along. Slow and steady.. This is great for me to learn, happy for anyone else to kick it off also, but it is getting worked on.
